internal/mq/msgstream/mq_factory.go

```diff
@@ -148,7 +148,7 @@ func (f *PmsFactory) NewMsgStreamDisposer(ctx context.Context) func([]string, st
 				if err != nil {
 					return err
 				}
-				msgstream.AsConsumer(channels, subname, mqwrapper.SubscriptionPositionUnknown)
+				msgstream.AsConsumer(channels, subname+"-close", mqwrapper.SubscriptionPositionUnknown)
```

Comment:
This `subname` shall be exactly the same so `Disposer` could do unsub for offline nodes

Reply:
But it will panic when the name is same
![image](https://user-images.githubusercontent.com/21985684/209770984-6b82a574-6787-4085-809c-0adaf50733e6.png)